packages/custom-ytsaurus-ui.example/src/server/configs/common.ts

```diff
@@ -21,6 +21,7 @@ const cfg: Partial<CustomAppConfig> = {
         reHashFromNodeVersion: '[^~]+~(?<hash>[^+]+)',
 
         reShortNameFromAddress: '(?<shortname>.*)(\\.[^.]+)(\\.yt\\.my\\.domain)',
+        reShortSystemAddressName: '(?<shortname>.*)(\\.[^.]+)(\\.yt\\.my\\.domain)',
```

Comment:
Isn't it better to use similar names for options?
`reShortNameFromMasterAddress`/`reShortNameSystemPage`

Reply:
As I remember `reShortNameFromAddress` is covered with a test. Is it possible to add similar one for the new option?

Reply:
I added it to the e2e configuration by analogy with the other 2.
But I don't see the point in covering a regular expression that changes in the configuration with a test.